tests/test_qubit_device_adjoint_jacobian.py

```diff
@@ -192,6 +192,64 @@ def test_gradient_gate_with_multiple_parameters(self, tol, dev):
         # the different methods agree
         assert np.allclose(grad_D, grad_F, atol=tol, rtol=0)
 
+    def test_return_expectation(self, tol, dev):
```

Comment:
I'm still not sure about the addition of `return_obs` - do you have a future use-case in mind?

Reply:
Easy enough to add in later if a use case arises, so I'll remove it for now.